### ISSUE TITLE
2018-2019 Fellowship Applications

### ISSUE THREAD

**Issue by tlasadea** [MozillaFoundation/CommsTeam#2]
Issue for outlining design work required for [this comms plan](https://docs.google.com/document/d/1STgAXUAb-4E1zKK5unMOh-EUUAYDHXFh0pzBovSMXBM/edit#).

**Comment by xmatthewx** [comment]
Hi @tlasadea - I took a quick look at the comms plans. Can you summarize here what you'll need design-wise?

**Comment by tlasadea** [comment]
No idea why I didn't also include [this](https://docs.google.com/presentation/d/1_valDol9T2TwBvSf-M-V68NhhG6d-j1w4sHServx_UE/edit#slide=id.g25275a8168_1_597), sorry @xmatthewx.

This project requires a bit of ideation time I think, as I know social graphics are required for e.g, but haven't been prescriptive in what that looks like.

**Comment by xmatthewx** [comment]
Thanks. Ideation time with designer, or before designer begins?

**Comment by tlasadea** [comment]
@xmatthewx with a designer sil vous plait

**Comment by xmatthewx** [comment]
## Desciption
 - Description: We're telling the world about the Fellowship's available at MoFo.
 - OKR 2.2, OKR 4.3

## DARCI
 - Driver: @tlasadea 
 - Accountable: @Lottarao 
 - Responsible: @taisdesouzalessa, @tlasadea , @KevZawacki  
 - Consulted: @auremoser, @rtufail , @community-impact
 - Informed: MoCo Comms, MoCo Social

**Comment by taisdesouzalessa** [comment]
@tlasadea please let me know when you want to talk about this :).

**Comment by tlasadea** [comment]
@taisdesouzalessa have sent you a meeting invite for thursday morning (8:30 - sorry it was the only available time :/)

**Comment by taisdesouzalessa** [comment]
Had a meeting with Temi to get the requirements and I will work on this task on the week of April 8th.

**Comment by taisdesouzalessa** [comment]
Hey @tlasadea - I did some studies today, they are all in **draft** shape (which means they are not refined yet, I just wanted to get some exploration of ideas and concepts done, I focused on quantity over quality for those studies) - once we choose a direction I can refine it and also experiment with some gifs. 

Could you please take a look at the directions and add your comments?
https://mozilla.invisionapp.com/boards/TD3LINZHG8X/

@xmatthewx those are in draft mode still but if you would like to check them and make comments, feel free to add them to InVision as well.

**Comment by taisdesouzalessa** [comment]
@tlasadea I added more options there, based on our feedback. 

I am staying away from pictures now and working with illustrations. Please let me know your thoughts and which track we should go for:
1. On Brand (green background)
2. Star Wars inspired option
3. Super Hero inspired option

I will also need the sizes once we define direction so I can think on animations on the final sizes.

**Comment by tlasadea** [comment]
I guess it makes sense to go with on brand (especially as I think those
were just designed this year?

**Comment by auremoser** [comment]
These look great @taisdesouzalessa :) 

I wanted to add too that the super-hero theme aligns with Ford's Public Interest Tech campaign (which the Fellows in Open Web fit under). They produced a series of videos about owf fellows which we cross-posted on medium: https://medium.com/tag/open-web-fellow/latest

Sid's is particularly super-hero-y (this was posted yesterday): https://twitter.com/FordFoundation/status/983736176619282433

I'm totally cool with "on brand" and defer to @tlasadea for definitive judgments but just wanted to throw this in the mix in case it's useful.

**Comment by taisdesouzalessa** [comment]
@auremoser good point on the super hero! I saw this campaign and that was was inspired me to go that route as well. Sid is a great super hero indeed!

Temi, regarding your question:
> guess it makes sense to go with on brand (especially as I think those were just designed this year?

Yep, they were designed this year :)

I made a GIF version for 2 on brand options. Aurelia and @tlasadea let me know what you think (the 2 first options of the board). If you think those options are good to go, I am ready to ship them - or if you need any modifications, let me know so I can work on them tomorrow.
https://mozilla.invisionapp.com/boards/TD3LINZHG8X/

I will also need the dimensions so I can adjust it for the social media channels we post. Currently the size is 1200x628px.

Thanks!

**Comment by tlasadea** [comment]
Hey @taisdesouzalessa, let's go with the brand versions. I like the superheros, but this is supposed to have longevity and thinking of longer ROI it makes sense to make it more evergreen.

I like the two GIF versions you have, but it would be nice to have one more iteration of the on brand version, so we can have three in totally? Especially because one of them is the current main graphic but just animated.

Re specs, we're only posting on Twitter, and they're below:

1024 x 512px, max JPEG, PNG file siz 5MB. Max GIF size 3MB.

**Comment by taisdesouzalessa** [comment]
@tlasadea thanks for the context :). I worked on 2 extra gifs today so we have 4 variations in total. Here they are, following the specs so they are ready to go!

In the last gif of this list I explored the connection between the areas, to reinforce the concept that we have an unified fellowship, even though we divide it in 3 areas. I like the connecting lines, I think they help to solidify the concept.

![on-brand-areas-animated-all-1](https://user-images.githubusercontent.com/22157921/38711426-754cb428-3e7a-11e8-9cda-b6a684456268.gif)
![on-brand-areas-animated-all-2](https://user-images.githubusercontent.com/22157921/38711427-756acf26-3e7a-11e8-9da5-d0b4b8fd8fcb.gif)
![on-brand-areas-animated-areas-1](https://user-images.githubusercontent.com/22157921/38711428-767a30d2-3e7a-11e8-8fe4-07fba4653cf4.gif)
![on-brand-areas-animated-areas-2](https://user-images.githubusercontent.com/22157921/38711429-7690d9fe-3e7a-11e8-9fd6-61e7662cffd3.gif)

InVision board is updated: https://mozilla.invisionapp.com/boards/TD3LINZHG8X/

@auremoser here are the final versions, in case you wanna check them out! :)

**Comment by rtufail** [comment]
Apologies for responding late - I was out sick earlier in the week. 

I don't want to use the terms Open Web, Tech Policy and Open Science because they push people back into the old way of thinking about the previous programs. If we need to, I would much prefer to use the terms policy professionals, open web activists and researchers.

**Comment by tlasadea** [comment]
@taisdesouzalessa can you change the names of the programs in the gifs based on Rizwan's feedback above?

**Comment by rtufail** [comment]
A further language nitpick. We are trying very hard to message that we have
one, single, unified fellowship program. That means internally and
externally we don't want to use the word programs to refer to our
fellowships. No more Open Web Fellows, Open Web Fellows Program or Tech
Policy Fellows and Tech Policy Fellowship Program. Just a single program
that invests in individuals - through fellowship and awards. People who
work with as fellows are known as Mozilla Fellows; they are open web
activists, researchers, policy professionals, artists, media-makers and
over time others. Thanks for your help in landing this message.

**Comment by tlasadea** [comment]
Sure Rizwan - we'll use the preferred language you mentioned before. Thanks!

**Comment by taisdesouzalessa** [comment]
@rtufail thanks for your feedback there. I'll use them the terms: `tech policy professionals`, `open web activists` and `scientists and researchers` - so we match exactly what we current have on the "Apply Now" page of the site. Does that work for you?

I'll update the language as recommended and also I'll do another change suggested by @xmatthewx: to integrate the Mozilla logo to the gifs. I will work on those today and post here!

Thanks for the collaboration, context and suggestions, team :).

**Comment by rtufail** [comment]
Thanks, Tais. That works very well.

**Comment by taisdesouzalessa** [comment]
Hey team,
here are the changes post-feedback:

1. Since I removed the areas from the option number one, I think a JPEG makes more sense

![on-brand-areas-all-1](https://user-images.githubusercontent.com/22157921/38757489-1c68e572-3f22-11e8-950a-6500c8a7db35.jpg)
![on-brand-areas-animated-all-2](https://user-images.githubusercontent.com/22157921/38757494-200ba7dc-3f22-11e8-99ce-f6dbaa927740.gif)
NEW! Updated!
![on-brand-areas-animated-areas-1](https://user-images.githubusercontent.com/22157921/38820642-4ad2276a-4153-11e8-8228-0cdb3c4e604b.gif)
NEW! Updated!
![on-brand-areas-animated-areas-2](https://user-images.githubusercontent.com/22157921/38820653-4ff36772-4153-11e8-96f4-2777f2b3238f.gif)



@rtufail @xmatthewx @tlasadea @auremoser

**Comment by auremoser** [comment]
Thanks @taisdesouzalessa !

Very cool. When will this material go out? 

Application closes on April 20th, are we allowed to use these on Monday? Could be cool to promote the last AMA on Wednesday with a promo-push on Monday: https://public.etherpad-mozilla.org/p/2018-fellowship-ama. I'll be tweeting about it but it would be delightful to collaborate on how we can loop these graphics into that promo.

**Comment by taisdesouzalessa** [comment]
Not sure when it is scheduled to go out @auremoser. I think @tlasadea knows, though :). If both you and Temi are okay with it, we could modify the copy of proposals 1 and 2 to include an AMA link there (we may need a short url). Or the AMA promo can be embedded on the Twitter copy. Just let me know how I can best support the initiative and I will be happy to help!

**Comment by tlasadea** [comment]
Thanks @taisdesouzalessa these look great! Can you pop the link to where these graphics are saved?
@auremoser these will be used to promote the application, yep.

**Comment by taisdesouzalessa** [comment]
@tlasadea - here is the Google Drive link: https://drive.google.com/open?id=119RAArqbfGB_TwRsOuiZH10dgsUfJS0F

**Comment by xmatthewx** [comment]
nice iterations @taisdesouzalessa! one small bit: I think the word Fellowships should be in black since it is part of a logo.

**Comment by auremoser** [comment]
Thanks so much folks! @tlasadea is it alright for me to promote the AMA on Wednesday using one of these graphics via a tweet today? Or is it best if it comes from \@mozilla

**Comment by taisdesouzalessa** [comment]
@xmatthewx I can change options 3 and 4 to accommodate that 👍

**Comment by taisdesouzalessa** [comment]
Options updated! Now "Fellowships" is in black for all options. @tlasadea @xmatthewx

**Comment by tlasadea** [comment]
Hey @auremoser I have something scheduled to go out at 11:42 PST time re Fellows apps, using the 2nd gif, but feel free to use one of the others !

**Comment by taisdesouzalessa** [comment]
This project has been shipped 🚢. Closing the issue. Feel free to reopen it if needed.
